data/data/bootstrap/files/usr/local/bin/kubelet.sh.template

```diff
@@ -11,6 +11,10 @@
     --runtime-request-timeout="${KUBELET_RUNTIME_REQUEST_TIMEOUT}" \
 {{- if .UseIPv6ForNodeIP }}
     --node-ip=:: \
+{{- end}}
+{{- if .BootstrapNodeIp }}
```

Comment:
UseIPv6ForNodeIP above is evaluted by the APIVIP
https://github.com/openshift/installer/blob/7cdf85d8df9a454c4de2297c5b5d4ae7b06fe96e/pkg/asset/ignition/bootstrap/common.go#L272

It seems that if the VIP is an IPv6 address and the bootstrapNodeIP is provided it will result in a malformed kubelet.sh with `--node-ip` flag provided twice.

Reply:
nope, look into the code. If you will provide bootstrap ip code will not set ipv6

Reply:
https://github.com/openshift/installer/pull/6042/files#diff-2a27cf9f710e504a0f8b3710fa10d24e403ff92e781cc8acbb425571427cc51bR278